docs/source/docfiles/markdown/QUICKSTART.md

```diff
@@ -60,6 +63,20 @@ bash ./resources/vm/setup_farmvibes_ai_vm.sh
 
 You might needed to restart your shell session once the script finishes.
 
+## Optional: Restore files with Git LFS
+
+In case you did not have Git LFS installed when cloning the repository, you will need to do so
+to restore the large files in the repository. Note that the last step
+["Installing software dependencies](#optional-installing-software-dependencies) already installs
+Git LFS.
+
+To restore the missing files, you can run the following command in the root of the repository:
+
+```shell
+git lfs install
+git lfs pull
+```
+
```

Comment:
Should we point out that clusters built without initializing LFS would need to be rebuilt? Perhaps adding a new entry on Troubleshooting?

Reply:
Done!